src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -85,9 +87,10 @@ class BankAccountStep extends React.Component {
     }
 
     toggleTerms() {
-        this.setState(prevState => ({
-            hasAcceptedTerms: !prevState.hasAcceptedTerms,
-        }));
+        this.setState((prevState) => {
+            this.debouncedUpdateReimbursementAccountDraft({hasAcceptedTerms: !prevState.hasAcceptedTerms});
+            return {hasAcceptedTerms: !prevState.hasAcceptedTerms};
```

Comment:
Same here, can we use a variable for this?

Reply:
Looks like this is still duplicated

Reply:
We are using two different keys for the component state vs `achData` + Onyx. I think that `hasAcceptedTerms` is better than `acceptTerms`, but changing the key in `achData` didn't seem worth it.

Reply:
+1

Reply:
Ah whoops missed that, yup this is fine then 👍

Reply:
We could do
```js
            const hasAcceptedTerms = !prevState.hasAcceptedTerms;
            updateReimbursementAccountDraft({acceptTerms: hasAcceptedTerms});
            return {hasAcceptedTerms};
```
although not sure if worth it 😄